src/instantx.cpp

```diff
@@ -983,22 +988,9 @@ bool CTxLockRequest::IsValid() const
         return false;
     }
 
-    CAmount nValueOut = tx->GetValueOut();
-
-    if(nValueIn - nValueOut < GetMinFee()) {
-        LogPrint("instantsend", "CTxLockRequest::IsValid -- did not include enough fees in transaction: fees=%d, tx=%s", nValueOut - nValueIn, ToString());
-        return false;
-    }
-
```

Comment:
You can't just drop it like that, should check if there is enough fee OR it's a "simple" tx.